conn.go

```diff
@@ -750,12 +792,12 @@ func (c *conn) writeProtoHeader(pID protoID) error {
 var keepaliveFrame = []byte{0x00, 0x00, 0x00, 0x08, 0x02, 0x00, 0x00, 0x00}
 
 // SendFrame is used by sessions and links to send frames across the network.
-func (c *conn) SendFrame(fr frames.Frame) error {
+func (c *Conn) SendFrame(fr frames.Frame) error {
```

Comment:
SendFrame -> sendFrame, yes?

Reply:
Good catch, also DeleteSession.

Reply:
I want to say I missed that one on purpose because I wanted you to build character.

Let's just say that.

Reply:
fixed